samples/place-autocomplete-basic-map/index.ts

```diff
@@ -12,50 +12,54 @@ const placeDetailsElement = document.querySelector(
   'gmp-place-details-compact',
 ) as any;
 const placeDetailsParent = placeDetailsElement.parentElement as HTMLElement;
-const mapDiv = document.getElementById('map-container') as HTMLElement;
-const center: google.maps.LatLngLiteral = { lat: 40.749933, lng: -73.98633 }; // New York City
-
+const gmpMapElement = document.querySelector('gmp-map') as any;
```

Comment:
s/any/google.maps.MapElement/

ditto others (generally, let's try to avoid `any`)